src/oop/user_defined_classes/hard/q3/Cupboard.java

```diff
@@ -0,0 +1,18 @@
+package oop.user_defined_classes.hard.q3;
+
+public class Cupboard
+{
+    private int pots;
+    private boolean anyClean;
+    public Cupboard()
+    {
+        pots = 0;
+        anyClean = false;
+    }
```

Comment:
I'm going to err on the side of not adding the constructor, mainly because the question works fine as is and adding a constructor will require a lot of reworking and take away time from other questions

Reply:
I actually think you should add the second constructor. We don't really have many examples of this happening.

Reply:
I'll toss it back to you and take my name off of it. Add me back as a reviewer when Kai approves it again please